volumesnapshotlocation.md

```diff
@@ -23,11 +23,16 @@ spec:
     # Optional.
     snapshotLocation: us-central1
 
+    # The project ID where snapshots should be stored.
```

Comment:
I'd like to clarify this comment to indicate that this **does not** enable creating the snapshots in a different project than where the volumes are; it's useful only if the project that contains the volumes is different than the project that contains the IAM account.